### PR TITLE
Download correct windows binary based on architecture

### PR DESCRIPTION
**Pull request by philipbalinov**
We currently assume all windows instances are amd64. Add a check for arm64 machines and install the corresponding binary.

### CODE REVIEW

File: tasks/windows.yml [hunk 1]
```diff
@@ -17,12 +17,19 @@
...
   when: pkg_version_url is undefined
 
-- name: Determine latest pkg version
+- name: Determine latest pkg version (amd64)
   ansible.builtin.set_fact:
     pkg_version_url: "{{ latest_jsondata.json | json_query(jmesquery) }}"
   vars:
     jmesquery: "files[*].filename | [?contains(@, 'amd64.msi')] | [0]"
-  when: pkg_version_url is undefined
+  when: ansible_architecture2 | lower != 'arm64' and pkg_version_url is undefined
```
Comment (czunker) [inline tasks/windows.yml:25]: What is `ansible_architecture2` ? I couldn't find it in the docs.

I only found `ansible_machine` and `ansible_architecture` .

Comment (philipbalinov) [reply tasks/windows.yml:25]: docs are useless (for windows), i ran ansible against an AWS EC2 windows instance, and a local arm instance, and this is what i got:

```
ansible -i hosts all -m setup -a 'filter=ansible_architecture' | grep ansible_arch
        "ansible_architecture": "64-bit",                                  # x86-64 ec2 instance
        "ansible_architecture2": "x86_64",                                 # x86-64 ec2 instance
        "ansible_architecture": "ARM 64-bit Processor",                    # arm64 instance
        "ansible_architecture2": "arm64",                                  # arm64 instance
```

Comment (czunker) [reply tasks/windows.yml:25]: Thanks for adding this.

**Review by czunker — APPROVED** [review]
Thanks @philipbalinov

**Pull request merged**